docker/setup-hosts.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+NODES=3
+
+
+## Create multiple hosts
+for N in $(seq 1 $NODES)
+do
+docker-machine create --driver virtualbox node$N
+done
+
+
+## Set up swarm
+### node1 as manager and others as workers
+SWARM_ADDR=$(docker-machine ip node1)
+
+docker-machine ssh node1 docker swarm init --advertise-addr $SWARM_ADDR
+
+SWARM_TOKEN=$(docker-machine ssh node1 docker swarm join-token -q worker)
+
+for N in $(seq 2 $NODES)
+do
+docker-machine ssh node$N docker swarm join --token $SWARM_TOKEN $SWARM_ADDR:2377
```

Comment:
I _think_ you can do
`docker swarm $(docker-machine config node$N) swarm join --token $SWARM_TOKEN $SWARM_ADDR:2377`

However, I had some issue where `docker-machine config` wasn't giving me the right paths to certificates and such. I think that was just a bug, not sure if you would hit the same issue.